src/Components/Facility/ConsultationDetails/index.tsx

```diff
@@ -126,6 +128,20 @@ export const ConsultationDetails = (props: any) => {
             data.symptoms_text = symptoms.join(", ");
           }
           setConsultationData(data);
+          const assetRes = await dispatch(
+            listAssetBeds({
+              bed: data?.current_bed?.bed_object?.id,
+            })
+          );
+          const isCameraAttachedRes =
+            assetRes.data.results.filter(
```

Comment:
If you want to check if one element atleast satisfies the condition, you can use `some` instead of `.filter(...).length > 1`

```suggestion
            assetRes.data.results.some(
```

Reply:
Done.

Reply:
Are you sure you've done this?

Reply:
> Are you sure you've done this?

Yeah sure